### DISCUSSION TITLE
More details on ports

### DISCUSSION THREAD

**Discussion by Rushaway** [aXenDeveloper/ips-app-advanced-serverlist#43] [Q&A]
Wich port need to be open for dont get (for a css/csgo sever)
["gq_online"]=>
    bool(false)
`array(1) {
  ["14.20.65.12:27025"]=>
  array(19) {
    ["gq_address"]=>
    string(14) "149.202.65.122"
    ["gq_dedicated"]=>
    NULL
    ["gq_gametype"]=>
    NULL
    ["gq_hostname"]=>
    NULL
    ["gq_joinlink"]=>
    string(37) "steam://connect/14.20.65.12:27025/"
    ["gq_mapname"]=>
    NULL
    ["gq_maxplayers"]=>
    NULL
    ["gq_mod"]=>
    NULL
    ["gq_name"]=>
    string(22) "Counter-Strike: Source"
    ["gq_numplayers"]=>
    NULL
    ["gq_online"]=>
    bool(false)
    ["gq_password"]=>
    NULL
    ["gq_port_client"]=>
    int(27072)
    ["gq_port_query"]=>
    int(10011)
    ["gq_protocol"]=>
    string(6) "source"
    ["gq_transport"]=>
    string(3) "udp"
    ["gq_type"]=>
    string(3) "css"
    ["players"]=>
    array(0) {
    }
    ["teams"]=>
    array(0) {
    }
  }
}`

**Comment by aXenDeveloper** [comment]
`14.20.65.12:27025` this is IP with your port.
